app/articles/_containers/SectionPosts/containers/PostCard/index.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import styles from "./index.module.scss";
+import Image from "next/image";
+import { Post } from '@/modules/business-types';
+import { useState, useEffect } from 'react';
+import cx from "clsx";
+
+type Props = {
+  className?: string;
+  style?: React.CSSProperties;
+  post: Post; 
+};
+
+export default function PostCard ({ className, style, post}: Props){
+  const formattedDate = new Date(post.date).toLocaleDateString('en-US', {
+    year: 'numeric', 
+    month: 'long', 
+    day: 'numeric', 
+  });
+
+  return (
+    <section  className={cx(styles.container, className)} style={style}>
+      <article className={styles.article}>
+          {/* Card image */}
+          <div className={styles.cardImg}>
+            {/* <Img 
```

Comment:
background image?

I have updated the background image on Wordpress. Make sure to update it correctly.

use Next Image for optimization.